<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>587</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>587</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>587</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#frame{
background-color:rgba(7, 19, 35, 240);
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="CrossButton">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: none;
	background-color: rgba(0,0,0,0);
   color: white
}
QPushButton:hover {
	background-color: rgba(0,125,236,255);
}
QPushButton:pressed {	
	background-color: rgba(0,0,0,0);
}</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MinimizeButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: none;
	background-color: rgba(0,0,0,0);
   color: white
}
QPushButton:hover {
	background-color: rgba(0,125,236,255);
}
QPushButton:pressed {	
	background-color: rgba(0,0,0,0);
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;DSA&lt;/span&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt; Lab 5&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>891</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#frame_2{
background-color:rgba(16, 30, 41, 240);
}


</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>-160</x>
       <y>70</y>
       <width>1071</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabBar::tab {
background-color:rgba(16, 30, 41, 240); 
color:rgba(16, 30, 41, 240); 
 }
QTabBar{
background-color:rgba(16, 30, 41, 240);
}

</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab2">
      <property name="styleSheet">
       <string notr="true">background-color:rgba(16, 30, 41, 240);
border:none</string>
      </property>
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white
</string>
       </property>
       <property name="text">
        <string>Student Detail:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="NameEdit">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>50</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(215, 215, 215)
</string>
       </property>
       <property name="text">
        <string>Student Name</string>
       </property>
       <property name="placeholderText">
        <string>Imported File</string>
       </property>
      </widget>
      <widget class="QPushButton" name="AddButton">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>250</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
   background-color:rgb(17, 30, 42);
	border: none;
   color: white;
    border-radius:10px;
}
QPushButton:hover {
	background-color: rgba(0,125,236,255);
}
QPushButton:pressed {	
	background-color: white;
   color : black
}</string>
       </property>
       <property name="text">
        <string>ADD Student</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>40</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white
</string>
       </property>
       <property name="text">
        <string>Table:</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="StuInfoTable">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>70</y>
         <width>501</width>
         <height>351</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget {

background-color:rgb(215, 215, 215);
color:black;


}</string>
       </property>
       <column>
        <property name="text">
         <string>Student Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>REG Number</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>GPA</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Number</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Univesity</string>
        </property>
       </column>
      </widget>
      <widget class="QLineEdit" name="RegEdit">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>90</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(215, 215, 215)
</string>
       </property>
       <property name="text">
        <string>Registration Number</string>
       </property>
       <property name="placeholderText">
        <string>Imported File</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="gpaEdit">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>130</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(215, 215, 215)
</string>
       </property>
       <property name="text">
        <string>GPA</string>
       </property>
       <property name="placeholderText">
        <string>Imported File</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="NumberEdit">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>170</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(215, 215, 215)
</string>
       </property>
       <property name="text">
        <string>Number</string>
       </property>
       <property name="placeholderText">
        <string>Imported File</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="uniEdit">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>210</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(215, 215, 215)
</string>
       </property>
       <property name="text">
        <string>University</string>
       </property>
       <property name="placeholderText">
        <string>Imported File</string>
       </property>
      </widget>
      <widget class="QPushButton" name="DeleteButton">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>350</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
   background-color:rgb(17, 30, 42);
	border: none;
   color: white;
    border-radius:10px;
}
QPushButton:hover {
	background-color: rgba(0,125,236,255);
}
QPushButton:pressed {	
	background-color: white;
   color : black
}</string>
       </property>
       <property name="text">
        <string>Delete Student</string>
       </property>
      </widget>
      <widget class="QPushButton" name="EditButton">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>300</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
   background-color:rgb(17, 30, 42);
	border: none;
   color: white;
    border-radius:10px;
}
QPushButton:hover {
	background-color: rgba(0,125,236,255);
}
QPushButton:pressed {	
	background-color: white;
   color : black
}</string>
       </property>
       <property name="text">
        <string>Edit Student</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>891</width>
       <height>101</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>23</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgba(0,125,236,255);
border-radius:10px;
background-color:rgb(16, 29, 41)</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;STUDENT&lt;/span&gt; CRUD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>891</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(16, 30, 41, 240);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
